hamyam/conditions.py

```diff
@@ -1,42 +1,17 @@
-import interactions
-import requests
-import base64
-import json
+import interactions, base64, requests
 
 def conditions(config):
 	# Return ham band conditions
 	# Takes: None
-	# Returns: interactions.py embed format
+	# Returns: interactions.py Image class object, ready to be sent via ctx.send
 
 	# Get current band conditions and convert it to base64
 	image = requests.get(config.config["CONDITIONS_URL"])
 	image_b64 = base64.b64encode(image.content)
+	# Now decode it from base64 to a BaseIO byte stream
+	image_bio = BytesIO(b64decode(re.sub("data:image/jpeg;base64", '', image_b64)))
```

Comment:
re.sub is undefined, assuming this is in reference to the regex library

Reply:
b64decode should be in form `base64.b64decode`

Reply:
BytesIO is undefined, `from io import BytesIO` is required.

Reply:
in line `image_bio = BytesIO(base64.b64decode(re.sub("data:image/jpeg;base64", '', image_b64)))`

`TypeError: cannot use a string pattern on a bytes-like object`